### PR TITLE
Actually stream points in the grpc server.

### PR DESCRIPTION
**Pull request by SirVer**
Instead of reading them all into memory.

### CODE REVIEW

File: point_viewer_grpc/src/bin/octree_server.rs [hunk 1]
```diff
@@ -69,48 +70,63 @@ impl proto_grpc::Octree for OctreeService {
...
+                }
+            });
         });
-        let f = resp.send_all(stream::iter_ok::<_, grpcio::Error>(replies))
+
+        // TODO(sirver): I did not figure out how to return meaningful errors. At least we return
+        // any error.
+        let rx = rx.map_err(|_| grpcio::Error::RemoteStopped);
+        let f = resp
+            .send_all(rx)
```
Comment (thomasschiwietz) [inline point_viewer_grpc/src/bin/octree_server.rs]: Could you explain this line to me, please?
The thread is sending reply messages through to channel back to receiver. Is the GRPC sent after all reply messages have been received or is it streaming?

Comment (SirVer) [reply point_viewer_grpc/src/bin/octree_server.rs]: added a couple of comments. ptal.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**